src/D2L.CodeStyle.Analyzers/ApiUsage/DangerousMemberUsages/DangerousMemberUsagesAnalyzer.cs

```diff
@@ -63,39 +63,54 @@ IImmutableSet<ISymbol> dangerousMethods
 				.GetSymbolInfo( invocation.Expression )
 				.Symbol;
 
-			if( methodSymbol.IsNullOrErrorType() ) {
+			if( !AnalyzePotentiallyDangerousMember( context, methodSymbol, auditedAttributeType, unauditedAttributeType, dangerousMethods ) ) {
 				return;
 			}
 
-			if( !IsDangerousMethodSymbol( methodSymbol, dangerousMethods ) ) {
-				return;
+			ReportDiagnostic( context, methodSymbol, Diagnostics.DangerousMethodsShouldBeAvoided );
+		}
+
+		private bool AnalyzePotentiallyDangerousMember(

```

Comment:
Broke this method out of the existing ```AnalyzeInnovation```, but does the same thing

Reply:
Please tell me you'll fix that typo as part of this? `Invocation`

Reply:
I can fix typeos after